spec/jobs/outlook/mass_destroy_event_job_spec.rb

```diff
@@ -28,26 +28,54 @@
     }
   end
 
-  before do
-    stub_request(:delete, "https://graph.microsoft.com/v1.0/me/Events/abc")
-      .with(headers: expected_headers)
-      .to_return(status: 204, body: "", headers: {})
-    stub_request(:delete, "https://graph.microsoft.com/v1.0/me/Events/def")
-      .with(headers: expected_headers)
-      .to_return(status: 204, body: "", headers: {})
-  end
+  context "when the events are deleted successfully" do
+    before do
+      stub_request(:delete, "https://graph.microsoft.com/v1.0/me/Events/abc")
+        .with(headers: expected_headers)
+        .to_return(status: 204, body: "", headers: {})
+      stub_request(:delete, "https://graph.microsoft.com/v1.0/me/Events/def")
+        .with(headers: expected_headers)
+        .to_return(status: 204, body: "", headers: {})
+    end
+
+    it "destroys the existing events in outlook" do
```

Comment:
```suggestion
    it "wipes the reference to the Outlook ID" do
```

Et je pense qu'on peut aussi avoir ce bloc dans le contexte `"when there is an error"`. :wink:

Reply:
J'étais tenté de laisser ce comportement en "undefined" en cas d'erreur, puisque je ne suis pas sûr qu'on veuille changer le comportement de `Outlook::DestroyEventJob`.